# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../iproute2/0002-include-limits.h.patch
# Copyright (C) 2013 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

include limits.h to get ULONG_MAX declared

------------------------------------------------------------------------------
fdb.c: In function 'fdb_modify':
fdb.c:242:30: error: 'ULONG_MAX' undeclared (first use in this function)
fdb.c:242:30: note: each undeclared identifier is reported only once for each function it appears in
------------------------------------------------------------------------------

detected when building in a musl based environment

--- ./bridge/fdb.c.orig	2013-08-29 02:03:20.878642825 +0200
+++ ./bridge/fdb.c	2013-08-29 02:04:09.963752113 +0200
@@ -20,6 +20,7 @@
 #include <linux/if_ether.h>
 #include <linux/neighbour.h>
 #include <string.h>
+#include <limits.h>
 
 #include "libnetlink.h"
 #include "br_common.h"
